README.md

```diff
@@ -88,10 +88,35 @@ weechat
 **NOTE:** If weechat is already running, the script can be loaded using ``/python load python/autoload/wee_slack.py``
 
 #### 4. Add your Slack API key(s)
+
+Log in to Slack:
+
+```
+/slack register
+```
+
+This command opens your browser and prompts you to authorize WeeChat with Slack.
```

Comment:
Well, it doesn't open the browser. It just prints a link, so maybe: `This command prints a link you should open in your browser [...]`

Reply:
Done.